lms/djangoapps/verify_student/views.py

```diff
@@ -1050,6 +1051,14 @@ def _submit_attempt(self, user, face_image, photo_id_image=None, initial_verific
         attempt.mark_ready()
         attempt.submit(copy_id_photo_from=initial_verification)
 
+        idv_update_signal.send(
+            sender='verify_student.submit_idv_attempt',
```

Comment:
We had put in sender to our original thinking to separate IDV from proctoring. Now that the events are completely different, do we need sender? I don't think any listeners should be told what exact function causes an event. Unless this is an edX event convention in which case 👍

Reply:
Fair point, I will update!